dask/highlevelgraph.py

```diff
@@ -1053,6 +1110,76 @@ def __dask_distributed_unpack__(hlg: dict) -> dict:
+        highlevelgraph_icon = """
+            <svg width="76" height="71" viewBox="0 0 76 71" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <circle cx="61.5" cy="36.5" r="13.5" fill="#F2F2F2" stroke="#1D1D1D" stroke-width="2"/>
+                <circle cx="14.5" cy="14.5" r="13.5" fill="#F2F2F2" stroke="#1D1D1D" stroke-width="2"/>
+                <circle cx="14.5" cy="56.5" r="13.5" fill="#F2F2F2" stroke="#1D1D1D" stroke-width="2"/>
+                <path d="M28 16L30.5 16C33.2614 16 35.5 18.2386 35.5 21L35.5 32.0001C35.5 34.7615 37.7386
+                    37.0001 40.5 37.0001L43 37.0001" stroke="black" stroke-width="1.5"/>
+                <path d="M40.5 37L40.5 37.75L40.5 37.75L40.5 37ZM35.5 42L36.25 42L35.5 42ZM35.5 52L34.75
+                    52L35.5 52ZM30.5 57L30.5 57.75L30.5 57ZM41.5001 36.25L40.5 36.25L40.5 37.75L41.5001
+                    37.75L41.5001 36.25ZM34.75 42L34.75 52L36.25 52L36.25 42L34.75 42ZM30.5 56.25L28.0001
+                    56.25L28.0001 57.75L30.5 57.75L30.5 56.25ZM34.75 52C34.75 54.3472 32.8472 56.25 30.5
+                    56.25L30.5 57.75C33.6756 57.75 36.25 55.1756 36.25 52L34.75 52ZM40.5 36.25C37.3244 36.25
+                    34.75 38.8243 34.75 42L36.25 42C36.25 39.6528 38.1528 37.75 40.5 37.75L40.5 36.25Z"
+                    fill="black"/>
+                <circle cx="28" cy="16" r="2.25" fill="#E5E5E5" stroke="black" stroke-width="1.5"/>
+                <circle cx="28" cy="57" r="2.25" fill="#E5E5E5" stroke="black" stroke-width="1.5"/>
+                <path d="M45.25 36.567C45.5833 36.7594 45.5833 37.2406 45.25 37.433L42.25 39.1651C41.9167
+                    39.3575 41.5 39.117 41.5 38.7321V35.2679C41.5 34.883 41.9167 34.6425 42.25 34.8349L45.25
+                    36.567Z" fill="#1D1D1D"/>
+            </svg>
+        """
+        layers_html = ""
+        for i, key in enumerate(self._toposort_layers()):
+            layer = self.layers[key]
+            layers_html += layer._repr_html_(
+                layer_index=f" {i}", highlevelgraph_key=key
+            )
+
+        html = f"""
+            <div>
+                <div>
+                    <div style="width: 52px; height: 52px; position: absolute;">
+                        {highlevelgraph_icon}
+                    </div>
+                    <div style="margin-left: 64px;">
+                        <h3 style="margin-bottom: 0px;">HighLevelGraph</h3>
+                        <p style="color: #5D5851; margin-bottom:0px;">
+                            {highlevelgraph_info}
+                        </p>
+                        {layers_html}
+                    </div>
+                </div>
+            </div>
+        """
+        return html
+
+
+def html_from_dict(info):
+    html = """<table style="width: 100%;">"""
+    suffix = """</table>"""
+    for key, val in info.items():
+        table_row = f"""
+          <tr>
+            <th style="text-align: left;">{key}</th>
```

Comment:
I think it's fairly safe to set the label to a fixed width:

```suggestion
            <th style="text-align: left; width: 150px;">{key}</th>
```

![image](https://user-images.githubusercontent.com/4806877/121542774-6800cf80-c9d6-11eb-9af6-4d2980ce9b6d.png)